src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -643,59 +659,88 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                     ) {
-                        return self.horzSelection.treatmentId;
+                        return self.horzSelection.genericAssayEntityId;
                     } else {
-                        return this.selectedTreatmentOption.value;
+                        return this.selectedGenericAssayOption.value;
                     }
                 } else {
                     return undefined;
                 }
             },
-            get selectedTreatmentOption() {
-                const treatmentOptions =
+            get selectedGenericAssayOption() {
+                const genericAssayOptions =
                     (vertical
-                        ? self.vertTreatmentOptions
-                        : self.horzTreatmentOptions.result) || [];
+                        ? self.vertGenericAssayOptions
+                        : self.horzGenericAssayOptions.result) || [];
                 if (
-                    this._selectedTreatmentOption === undefined &&
-                    treatmentOptions.length
+                    this._selectedGenericAssayOption === undefined &&
+                    genericAssayOptions.length
                 ) {
-                    // select default if _selectedTreatmentOption is undefined and there are treatments to choose from
-                    return treatmentOptions[0];
+                    // select default if _selectedGenericAssayOption is undefined and there are generic assay entities to choose from
+                    return genericAssayOptions[0];
                 } else if (
                     vertical &&
-                    this._selectedTreatmentOption &&
-                    this._selectedTreatmentOption.value ===
+                    this._selectedGenericAssayOption &&
+                    this._selectedGenericAssayOption.value ===
                         SAME_SELECTED_OPTION_STRING_VALUE &&
                     self.horzSelection.dataType === CLIN_ATTR_DATA_TYPE
                 ) {
                     // if vertical gene set option is "same as horizontal", and horizontal is clinical, then use the actual
                     //      gene set option value instead of "Same gene" option value, because that would be slightly weird UX
-                    return self.horzSelection.selectedTreatmentOption;
+                    return self.horzSelection.selectedGenericAssayOption;
                 } else {
                     // otherwise, return stored value for this variable
-                    return this._selectedTreatmentOption;
+                    return this._selectedGenericAssayOption;
+                }
+            },
+            set selectedGenericAssayOption(o: any) {
+                this._selectedGenericAssayOption = o;
+            },
+            get isGenericAssayType() {
+                if (!self.dataTypeOptions.isComplete) {
+                    // if there are no options to select a default from, then return false as default
```

Comment:
the comment seems out of date - you're not returning false, right?

Reply:
no, I am not. I will update this.